ts/sagas/wallet/pagopaApis.ts

```diff
@@ -478,10 +482,15 @@ export function* paymentVerificaRequestHandler(
   action: ActionType<typeof paymentVerifica["request"]>
 ) {
   try {
+    const isPagoPATestEnabled: ReturnType<
+      typeof isPagoPATestEnabledSelector
+    > = yield select<PersistedPreferencesState>(isPagoPATestEnabledSelector);
```

Comment:
not related to the bug but this and the following should be

```suggestion
    > = yield select<GlobalState>(isPagoPATestEnabledSelector);
```